src/lib/holocene/orderable-list/orderable-list.stories.svelte

```diff
@@ -77,6 +80,9 @@
         static={false}
         label={item.label}
         pinned={item.pinned}
+        moveUpButtonLabel="Move Up"
+        moveDownButtonLabel="Move Down"
+        removeButtonLabel="Remove"
```

Comment:
If you lack the willpower to make this translations... I won't blame you one bit.

Reply:
Whelp nevermind, it's in a story.